### PR TITLE
added release notes for 10.15.0.327

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device Management</td>
+<td>On the <b>Services</b> tab in the device details, the filter for the "Status" column did not work properly and returned no results when filtering was applied. This is now fixed.</td>
```
Comment (rssag) [inline content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md:25]:
```suggestion
<td>On the <b>Services</b> tab in the device details, the filter for the "Status" column did not work properly and returned no results when a filter was applied. This is now fixed.</td>
```

File: content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<td>DM-2002</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>During the LWM2M client registration, if the agent failed to find the registered device from the database due to any communication problem, then an internal object related to the device managed object was automatically removed by the agent. Thereafter, a LWM2M client always failed to connect. With the current fix, now the internal component is only removed if the actual device managed object does not exist in the database.</td>
```
Comment (rssag) [inline content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md:31]:
```suggestion
<td>During the LWM2M client registration, if the agent failed to find the registered device from the database due to a communication problem, an internal object related to the device managed object was automatically removed by the agent. Afterwards, LWM2M clients always failed to connect. This is now fixed and the internal object is only removed if the actual device managed object does not exist in the database.</td>
```

File: content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+<td>DM-2131</td>
+</tr>
+
+<tr>
+<td>Loriot</td>
+<td>Events created by the Loriot agent when sending uplink requests are now created in a microservice context without the application key. As a result, the event creation calls will now be considered as device requests.</td>
```
Comment (rssag) [inline content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md:37]:
```suggestion
<td>Events created by the Loriot agent when sending uplink requests are now created in a microservice context without the application key. As a result, the event creation calls are now considered device requests.</td>
```

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+<tr>
+<td>LWM2M</td>
+<td>During the LWM2M client registration, if the agent failed to find the registered device from the database due to a communication problem, an internal object related to the device managed object was automatically removed by the agent. Afterwards, LWM2M clients always failed to connect. This is now fixed and the internal object is only removed if the actual device managed object does not exist in the database.</td>
+<td>DM-2131</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>Due to a limitation of the Leshan library used by the LWM2M agent, only device registration messages were supported where the protocol version is defined to the LWM2M object. The LWM2M agent has now been extended to also accept LWM2M registration messages when the LWM2M version is attached to the LWM2M object instance.</td>
```
Comment (rssag) [inline content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md:37]:
```suggestion
<td>Due to a limitation of the Leshan library used by the LWM2M agent, only device registration messages were supported where the LWM2M version is attached to the LWM2M object. The LWM2M agent has now been extended to also accept LWM2M registration messages where the LWM2M version is attached to the LWM2M object instance.</td>
```

confusing still - what is the difference between an object and the object instance?
from basic programming: the object is the instance of a class

Comment (BeateRixen) [reply content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md:37]: @Oezge-Akin Can you please have one more look?

**Review by rssag — APPROVED** [review]

**Pull request merged**